src/test/java/net/minecraftforge/debug/DynBucketTest.java

```diff
@@ -335,14 +338,14 @@ public void onDataPacket(NetworkManager net, SPacketUpdateTileEntity pkt) {
         }
 
         @Override
-        public boolean hasCapability(@Nonnull Capability<?> capability, @Nullable EnumFacing facing)
```

Comment:
With this PR I added a package annotation to minecraftforge/debug/ so that fields and returns values are @Nonnull by default, to cut down on the extreme spam of @Nonnull everywhere that's required.